.gitpod.yml

```diff
@@ -22,8 +22,9 @@ tasks:
           -e  "s@development\.yml@gitpod\.yml@" \
           config/env.development > .env
     init: |
-      pnpm install
-      docker-compose --env-file .env build $SERVICES
+      pnpm install & \
+      docker-compose --env-file .env build $SERVICES & \
```

Comment:
I wonder if we'll have any race conditions with this (thinking of the pnpm lock file and dependencies service).  cc @JerryHue

Reply:
It should be ok as the dep collection script is run on `pre-version`. 
What do u think @JerryHue?